### ISSUE TITLE
Cameras?

### ISSUE THREAD

**Issue by sambarlick** [tuya/tuya-home-assistant#64]
Are Cameras on the list for supported devices in the future?

I have some Tuya based cameras that I want to be intergrated, even if it's just as a motion sensor, but ideally with the feed.

Thanks,
Sam

**Comment by jkottier** [comment]
Indeed, basic (doorbell) camera support with at least movement and doorbell pressed notifications would be highly appreciated.

**Comment by tsutsuku** [comment]
We will consider it.

**Comment by jkottier** [comment]
I'm currently using a workaround to achieve this using a (virtual) Tuya switch and a Smart Life "Smart" automation to capture doorbell that turns on the Switch on a doorbell event. De switch events can be caucht in Home Assistant using an automation.

From that point on you can trigger anything using your Home Assistant.

**Comment by gomble** [comment]
Camera Support would be nice !!!

**Comment by kevinderek** [comment]
+1

**Comment by brunokelo** [comment]
+1

**Comment by cq74** [comment]
Support for cameras would be really great

**Comment by WenderGalan** [comment]
I would love to have support for cameras, I have several Tuya cameras but no integration.

**Comment by brunokelo** [comment]
+1

**Comment by dpomnean** [comment]
+1

**Comment by bafplus** [comment]
Same here, i have multiple mini 14S camera's, would love to be able to use them in HA, or beter yet a direct way to acces the streams but thats out of the scope of Tuya v2 i guess :-)

**Comment by naijaping** [comment]
Currently playing with Home Assistant 2021.11, Although there is now support for doorbell camera. Unfortunately no    Motion and Doorbell pressed  entity. all I have is  camera entity, camera battery entity, camera flip, night vision and watermark entity.
hope this will be looked at in the next update .

thanks

**Comment by kevinderek** [comment]
??

**Comment by frenck** [comment]
@kevinderek ?

**Comment by bafplus** [comment]
@kevinderek ??

**Comment by bafplus** [comment]
> @frenck You indicate that this would have been added in version 2021.11, but as naijaping says this is not the case. Or do we need to add this somewhere separate???

Judging by reactions on FB, it seems that not all, or even most doorbell cameraś are not fully implemented and are missing the camerafeed function. All other entities are present except the stream entity.

**Comment by frenck** [comment]
Cool? Dunno. If you think there is a bug, raise an issue. Not all 400K+ different Tuya devices are implemented. Feel free to contribute additional device support!

**Comment by bafplus** [comment]
> Currently playing with Home Assistant 2021.11, Although there is now support for doorbell camera. Unfortunately no Motion and Doorbell pressed entity. all I have is camera entity, camera battery entity, camera flip, night vision and watermark entity. hope this will be looked at in the next update .
> 
> thanks

As @frenck mentioned, please open a seperate issue for your camera. Dont forget to put in as much info as possible like brand and model of the camera, what entities do you get and what not etc.

**Comment by kevinderek** [comment]
Ok I get that, but I already did that.
https://github.com/tuya/tuya-home-assistant/issues/254


but there I had the same answer. 
This has been addressed and added/fixed with the Home Assistant Core version of the integration and is pending release with Home Assistant 2021.11.

**Comment by frenck** [comment]
The topic asked for 

![image](https://user-images.githubusercontent.com/195327/141256916-03f867cf-41e3-4dab-9bfb-b7d97a4579e9.png)

The camera has been added as requested. As a matter of fact, I bought that device (from the specific brand) to make the camera work.
